config/integrations/github_graphql.rb

```diff
@@ -34,7 +34,7 @@ def vulnerability_alerts(repos: nil)
 
       query_string = repos.sort.map { |r| "repo:#{r}" }.join(" ")
       result = client.query(VulnerabilityAlertQuery, variables: { "queryString" => query_string  })
-      result.data.search.edges.each_with_object([]) do |edge, alerts|
+      (result.data&.search&.edges || []).each_with_object([]) do |edge, alerts|
```

Comment:
This looks fine. It might be a little more clear to pull out a method or use a variable rather than lean on `||`, but I don't think it's much (if any) better.